.circleci/config.yml

```diff
@@ -110,11 +110,16 @@ jobs:
             chmod +x ~/docker-compose
             sudo mv ~/docker-compose /usr/local/bin/docker-compose
             docker-compose --version
+      - run:
+          name: Set hosts
+          command: |
+            echo 127.0.0.1 olympia.test | sudo tee -a /etc/hosts
+            cat /etc/hosts
       - run:
           name: Start container, verify it's running and start tests
           environment:
-            UITEST_FXA_EMAIL: uitest-$(uuid)@restmail.net
             MOZ_HEADLESS: 1
+            PYTEST_ADDOPTS: "--reruns 2"
```

Comment:
Is this because there are intermittents, or do some tests fail the first time?

Reply:
2 main reasons. 
1. Sometimes there is just a bit of slowness and it effects some tests more than other, i.e.: the devhub upload test. It sometimes times out because the addon doesn't get uploaded.

2. Geckodriver sometimes cannot start marionette on a port and we get an error that marionette couldn't find an open port. 

Note these are ONLY for CI. A local test run does not have reruns.